cfgov/cfgov/settings/local.py

```diff
@@ -0,0 +1,14 @@
+from .base import *
+import v1
+
+DEBUG = True
+
+ALLOWED_HOSTS = ['*']
+
+STATIC_URL = '/static/'
+
+SHEER_SITES = [os.path.dirname(v1.__file__) + '/jinja2/v1']
```

Comment:
If we're still using /dist/ then that has to be a sheer site for it to load up the templates.